scripts/steps/build-api.js

```diff
@@ -110,6 +104,105 @@ const UNWRAP_UTILITIES = new Set([
   'Readonly'
 ])
 
+const EXCLUDED_SUBMENU_KINDS = new Set(['Type alias', 'Interface', 'Namespace'])
+const EXCLUDED_TREE_KINDS = new Set(['Namespace'])
+const CORE_ENTITIES = new Set([
+  'BaseNode',
+  'ClientNode',
+  'Connection',
+  'Log',
+  'LogStore',
+  'Reconnect',
+  'ServerConnection',
+  'ServerNode'
+])
+const CLIENT_ENTITIES = new Set([
+  'Client',
+  'CrossTabClient',
+  'badge',
+  'buildNewSyncMap',
+  'changeSyncMap',
+  'changeSyncMapById',
+  'confirm',
+  'createFilter',
+  'createSyncMap',
+  'defineSyncMap',
+  'deleteSyncMap',
+  'deleteSyncMapById',
+  'encryptActions',
+  'request'
+])
+const SERVER_ENTITIES = new Set([
+  'BaseServer',
+  'ChannelContext',
+  'Context',
+  'ResponseError',
+  'Server',
+  'ServerClient',
+  'del',
+  'get',
+  'patch',
+  'post',
+  'put',
+  'request'
+])
+const STATE_ENTITIES = new Set([
+  'cleanStores',
+  'createDerived',
+  'createMap',
+  'createPersistent',
+  'createRouter',
+  'createStore',
+  'getPagePath',
+  'getValue',
+  'openPage'
+])
+const TEST_ENTITIES = new Set([
+  'TestClient',
+  'TestLog',
+  'TestPair',
+  'TestServer',
+  'TestTime',
+  'emptyInTest',
+  'prepareForTest',
+  'eachStoreCheck'
```

Comment:
It is a rare tool. I expect that we have only 5-10 log stores even if Logux became very popular.

Reply:
move it to 'Other'